NoteFind/Views/OfflineGridCell.h

```diff
@@ -0,0 +1,16 @@
+//
+//  OfflineGridCell.h
+//  NoteFind
+//
+//  Created by Khloe Wright on 8/4/22.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OfflineGridCell : UICollectionViewCell
+@property (strong, nonatomic) IBOutlet UIImageView *notePost;
```

Comment:
notePostImage seems like a better name since the image view isn't the whole post?